src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/VisualBasicSymbolMatcher.vb

```diff
@@ -510,7 +510,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit
             End Function
 
             Private Function AreNamedTypesEqual(type As NamedTypeSymbol, other As NamedTypeSymbol) As Boolean
```

Comment:
Comments for C# version of this function is applicable here as well. In addition, it doesn't look like this function makes any distinction between Anonymous Types and Anonymous Delegates. It feel like the distinction should be made.